signup.html

```diff
@@ -16,27 +16,30 @@
     <div class="content">
       <div class="head-img">
         <a class="login-link" href="/">
-          <img class="pandaMarketImg" src="img/logo.png" />
+          <img class="pandaMarketImg" src="img/logoHead.png" />
+          <img class="pandaMarketImg-name" src="img/logoName.png" />
         </a>
       </div>
       <form action="/signup" method="post">
         <div class="input-content">
           <label for="userName" class="userName-label">이메일</label>
           <input
             type="email"
-            name="userEmail"
+            name="userName"
             id="userName"
             class="userName"
             placeholder="이메일을 입력해주세요"
           />
+          <div id="error-message"></div>
           <label for="nicname" class="nicname-label">닉네임</label>
           <input
             type="text"
-            name="userName"
+            name="nicName"
```

Comment:
❗️ 수정요청
오타이신 것 같아요~

```suggestion
            name="nickname"
```